src/lib/orionld/mongoCppLegacy/mongoCppLegacyEntityOperationsUpsert.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+*
+* Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Gabriel Quaresma
+*/
+#include "mongo/client/dbclient.h"                                         // MongoDB C++ Client Legacy Driver
+
+extern "C"
+{
+#include "kjson/KjNode.h"                                                  // KjNode
+#include "kjson/kjRender.h"                                                // kjRender - TMP
+#include "kjson/kjBuilder.h"                                               // kjArray, ...
+}
+
+#include "logMsg/logMsg.h"                                                 // LM_*
+#include "logMsg/traceLevels.h"                                            // Lmt*
+
+#include <vector>
```

Comment:
Standard include file go before "our" include files.

1. C headers
2. C++ headers
3. Headers of externalk libraries, like m ongo/MHD, etc
4. Our own header files

Make style should complain about this ...